py_gtfs_rt_ingestion/pyproject.toml

```diff
@@ -8,9 +8,9 @@ authors = ["MBTA CTD <developer@mbta.com>",
 [tool.poetry.dependencies]
 python = "^3.9"
 pyarrow = "^8.0.0"
-boto3 = "^1.23.3"
 
 [tool.poetry.dev-dependencies]
+boto3 = "^1.23.3"
```

Comment:
eek. this is creeped in there. i moved it so that boto3 isn't included in zip files i make for builds that are deployed to lambda. i can take this out if you like.

Reply:
Makes sense to me.